pandaserver/dataservice/setupper_atlas_plugin.py

```diff
@@ -847,7 +846,7 @@ def subscribe_dispatch_data_block(self) -> None:
                 # destination
                 tmp_dst_id = job.computingSite
                 tmp_site_spec = self.site_mapper.getSite(job.computingSite)
-                scope_tmp_site_input, _ = select_scope(tmp_site_spec, job.prodSourceLabel, job.job_label)
+                scope_tmp_site_input, scope_tmp_site_output = select_scope(tmp_site_spec, job.prodSourceLabel, job.job_label)
```

Comment:
when a variable is not used later, but you need a placeholder for the return of a function, it's common to use "_" to throw some garbage in it. I would not replace these, unless you will use scope_tmp_site_output later.

Reply:
It is used later on, I had a bug apparently in my previous refactored code